### DISCUSSION TITLE
Can it be deployed on the internal network?

### DISCUSSION THREAD

**Discussion by Gaslox** [projectdiscovery/interactsh#62] [Q&A] [answered]
➜  interactsh-client git:(main) ✗ ./interactsh-client -url https://10.10.123.237

    _       __                       __       __
   (_)___  / /____  _________ ______/ /______/ /_
  / / __ \/ __/ _ \/ ___/ __ '/ ___/ __/ ___/ __ \
 / / / / / /_/  __/ /  / /_/ / /__/ /_(__  ) / / /
/_/_/ /_/\__/\___/_/   \__,_/\___/\__/____/_/ /_/ v0.0.3

		projectdiscovery.io

[INF] Listing 1 URL for OOB Testing
[INF] c43v1fecie6h4kfllrvgcrd9bxeyyyyyn.10.10.123.237

How to use httplog ？ 

http://10.10.123.237/c43v1fecie6h4kfllrvgcrd9bxeyyyyyn 

Is this supported ？

**Comment by ehsandeep** [comment, marked as answer]
@huigou90 you need a self-hosted domain name to configure your own version of interactsh-server, you can find the install instructions here - https://github.com/projectdiscovery/interactsh#setting-up-self-hosted-instance
